packages/server/lib/makeDataContext.ts

```diff
@@ -157,16 +157,16 @@ export function makeDataContext (options: MakeDataContextOptions): DataContext {
         })
       },
       showItemInFolder (folder: string) {
-        require('electron').shell.showItemInFolder(folder)
+        electron.shell.showItemInFolder(folder)
       },
       showOpenDialog (props: OpenDialogOptions) {
-        return require('electron').dialog.showOpenDialog(props)
+        return electron.dialog.showOpenDialog(props)
       },
       showSaveDialog (window: BrowserWindow, props: SaveDialogOptions) {
-        return require('electron').dialog.showSaveDialog(window, props)
+        return electron.dialog.showSaveDialog(window, props)
       },
       copyTextToClipboard (text: string) {
-        require('electron').clipboard.writeText(text)
+        electron.clipboard.writeText(text)
       },
       // These instances of JIT requiring gui/windows can be removed
       // once https://github.com/cypress-io/cypress/issues/21236 is fixed
```

Comment:
I think it should be possible to remove these guys as well

Reply:
Good call, I should have seen that that comment. I'll get these updated too.